pkg/apis/policy/validation/validation_test.go

```diff
@@ -389,6 +389,14 @@ func TestValidatePodSecurityPolicy(t *testing.T) {
 	invalidProcMount := validPSP()
 	invalidProcMount.Spec.AllowedProcMountTypes = []api.ProcMountType{api.ProcMountType("bogus")}
 
+	emptyAllowedCSIDrivers := validPSP()
```

Comment:
I think validation should only happen when the feature is enabled. @liggitt can you confirm?

Reply:
Will wait for @liggitt  response here.

Reply:
if the feature is disabled:
* new pods will have their CSI `pod.spec.volumes[*].csi` field cleared, so validation wouldn't be applicable
* old pods without the CSI field set cannot update to set it, so validation is not applicable there either
* old pods with the CSI field set would preserve it, so PSP validation could affect behavior there

if a user updates an existing pod (changes the image, etc) which has an inline CSI volume (which means the pod was created when the feature gate was on), and there's a PSP with CSI fields (which also must have been created when the feature gate was on), would you expect the policy to still enforce the CSI fields. I think I would.